src/main/java/frc/robot/subsystems/hood/Hood.java

```diff
@@ -0,0 +1,68 @@
+package frc.robot.subsystems.hood;
+
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import static frc.robot.Ports.Hood.SOLENOID;
+
+public class Hood extends SubsystemBase {
+    private static Hood INSTANCE;
+    private final Solenoid angleChanger = new Solenoid(PneumaticsModuleType.CTREPCM, SOLENOID);
+
+    private Hood() {
+    }
+
+    public enum Mode {
+        ShortDistance(true),
+        LongDistance(false);
```

Comment:
I don't think true and false fit here. Why not angles?

Reply:
It represents the 2 modes of the solenoid. true - open, false - closed.

Reply:
A boolean doesn't fit here IMO. It's not like in the intake where "retracted" is the opposite of "un-retracted" (and even there I would use an enum). Here, 54° is not the opposite of 69°. They are just two different options. What would you do if you had three options?

Reply:
There can't be 3 options because the only part used in the subsystem is a single solenoid.